applications/asset_tracker/README.rst

```diff
@@ -121,33 +142,32 @@ After programming the application and all prerequisites to your board, test the
 1. Connect the board to the computer using a USB cable.
    The board is assigned a COM port (Windows) or ttyACM device (Linux), which is visible in the Device Manager.
 #. Connect to the board with a terminal emulator, for example, LTE Link Monitor.
-#. Open a web browser and navigate to https://nrfcloud.com/.
-   Follow the instructions to set up your account and add an LTE device.
-   A pattern of switch and button actions is displayed.
+#. Reset the board.
 #. Observe in the terminal window that the board starts up in the Secure Partition Manager and that the application starts.
    This is indicated by output similar to the following lines::
 
       SPM: prepare to jump to Non-Secure image
       ***** Booting Zephyr OS v1.13.99 *****
       Application started
 
-#. Observe that LED 3 starts blinking as the LTE link is established. This may take several minutes.
-#. Observe in the terminal window that the connection to nRF Cloud is established.
+#. Observe in the terminal window that the connection to nRF Cloud is established. This may take several minutes.
```

Comment:
Should we keep a mention of the LTE link here? `Observe in the terminal window that LTE link and connection to nRF cloud is established`

Reply:
Since the LTE link notification is followed by more scrolling text quite soon, I don't think it makes sense to ask the user to look for it.